src/communication/communication.gateway.ts

```diff
@@ -99,6 +103,31 @@ export class CommunicationGateway implements OnGatewayDisconnect {
     }
   }
 
+  /**
+   * Send veriff verification to a specific user by DID
+   * This method is called from VeriffService to emit verification results
+   * @param recipientDid the recipient DID
+   * @param payload the verification payload
+   * @returns boolean indicating success
+   */
+  sendVeriffVerificationToDid(recipientDid: string, payload: string): boolean {
+    try {
+      const recipientSocketId = this.usersService.getLoggedInUser(recipientDid);
+
+      if (!recipientSocketId) {
+        this.logger.warn(`User ${recipientDid} is not connected`);
+        return false;
+      }
+
+      // Use server.to() to emit to specific socket ID
+      this.server.to(recipientSocketId).emit('veriffVerification', payload);
```

Comment:
should not need this property:
see `src/communication/communication.gateway.ts` Line 99 you can use `sthis.userService()` to send message to the user